src/drive/web/modules/upload/index.js

```diff
@@ -236,6 +240,32 @@ export const overwriteFile = async (client, file, path) => {
   return resp.data
 }
 
+export const uploadFilesFromNative = (
+  files,
+  folderId,
+  uploadFilesSuccessCallback
+) => async dispatch => {
+  dispatch({
+    type: ADD_TO_UPLOAD_QUEUE,
+    files: files
+  })
+  //!TODO Promise.All
+  for (var i = 0; i < files.length; ++i) {
```

Comment:
Nit:

```suggestion
  for (const i = 0; i < files.length; ++i) {
```

`var` is so old school 😄